Automattic/wp-calypso
# BackupBrowser: Add Download selected files functionality

## Proposed Changes

* Adds the ability to do a granular download
* We still have PRs open for the API functionality to fully test this process

## Testing Instructions



* Visit the Backup Browser locally or in Live
* If using Live Branches, add `?flags=jetpack/backup-granular` to the URL
* Select some files, and select 'Downloaded selected files' from the top
* Once the files are downloaded, ensure the proper files exist in the download
* You should also receive and email with the download link to allow users to access the file later or if they left the screen

## Pre-merge Checklist
